src/SlashCommands.js

```diff
@@ -271,7 +271,22 @@ var commands = {
             }
         }
         return reject(this.getUsage());
-    })
+    }),
+    status: new Command("status", "<online|offline|unavailable> [<status message>]", function(roomId, args) {
+      if (args) {
+          let matches = args.match(/^(online|offline|unavailable)( +(.*))?$/);
```

Comment:
perhaps you want `([^ ].*)` for the innermost pattern, to make sure that the status message is non-empty.